weblate/utils/stats.py

```diff
@@ -133,6 +133,40 @@ def prefetch_stats(queryset):
     return result
 
 
+def get_non_glossary_stats(stats_obj) -> dict[str, int]:
```

Comment:
Please always include type annotations for new code:

```suggestion
def get_non_glossary_stats(stats_obj : ProjectLanguageStats | ProjectStats | GlobalStats) -> dict[str, int]:
```

Reply:
Understood.